guide/13-managing-arcgis-applications/enterprise-sites-guide.ipynb

```diff
@@ -0,0 +1,578 @@
+{
```

Comment:
<p>We will now delete the red text card from the page that reads "Where we're going". To do this, we fetch the layout of the page, delete the card from the layout, and publish the changes. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1450/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>